static/js/stream_edit.js

```diff
@@ -356,10 +356,11 @@ function stream_setting_clicked(e) {
 
     // A hack.  Don't change the state of the checkbox if we
     // clicked on the checkbox itself.
-    if (checkbox[0] !== e.target) {
+    // git log 88f3325970b
```

Comment:
What's the story behind commenting this out and referencing the commit hash?

Reply:
Oh, commented this out as the checkbox in my new added table does not get updated. (I could set the checked property in another function for the new table.) 
Sorry, just trying to figure out this block. (but the checkboxes in stream edit page seems to be working fine on comment out this block)